### ISSUE TITLE
Legacy plugin dynmap v3.1-beta7-449 does not specify an api-version.

### ISSUE THREAD

**Issue by TukaWarrior** [webbukkit/dynmap#3299]
The map isn't working with my set port and the server adress. I am running a Paper server which should be able to run spigot and bukkit plugins. 
I read that I need to set an API version in a plugins.yml file. But I can't find such a file anywhere on the server. What do I have to do exactly?

**Comment by stepech** [comment]
hey, do you have a host, or do you host on your own computer?
That api version is not mistake on your side, it's in plugin's code and it should't cause any issues.

**Comment by TukaWarrior** [comment]
Hi, thanks for the reply and your time. Luckily, I found a solution. Sort of. 
I pay a host. MCProHosting. 
I managed to partially solve the issue. But a new one occured. 
I have no idea about coding etc but I managed to find the plugins.yml file inside the .jar file of the plugin. I added api-version = 1.16 and it worked. :D The server console sais that the plugin is working correctly.  Now the new problem that occured seems to be a problem with MCProHosting itself. 
I opened a port on the server (the same that I set as webserver port in the plugins config file), so people can access the map in their browser. But as soon as I tell MCProHosting to open any port besides the main port for the server, everything crashes and the server won't start up again, even when I undo all the stuff. I basically have to reinstall the server.
